doc/developer/workflow.rst

```diff
@@ -283,7 +283,9 @@ Pre-submission Checklist
    -  ``make test``
 
```

Comment:
Suggest adding something like: 
  - The addition of automated testing to cover any pull request is encouraged.

Reply:
added